### PR TITLE
reformatted mesheryctl version output

### LINKED ISSUE #765: [mesheryctl] GitSHA in `mesheryctl version` should specify client vs server
**Current Behavior**
`mesheryctl version` returns:
```
Client Version: 0.3.11
Server Version: 0.3.11
GitSHA:2f20e18af829df593b94a670fa70c62d80a5344e
```
**Desired Behavior**
The GitSHA in this response should be identified as being related to the client (as opposed to the server). Its indistinguishable currently.

### PR DESCRIPTION
**Pull request by ashiscs**
fixes #765 
Signed-off-by: ashiscs <ashissingh640@gmail.com>

### CODE REVIEW

File: mesheryctl/cmd/version.go [hunk 1]
```diff
@@ -35,9 +35,8 @@ var versionCmd = &cobra.Command{
 	Short: "Version of mesheryctl",
 	Long:  `Version of Meshery command line client - mesheryctl.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Info("Client Version: ", BuildClient)
+		log.Info("Client Version: %v \t  GitSHA: %v", BuildClient, CommitSHA)
```
Comment (anandrkskd) [inline mesheryctl/cmd/version.go:38]: `log.info` should be replaced with `log.infof`
For safe practice, please always try it on local by creating a binary using `make` command.

Comment (ashiscs) [reply mesheryctl/cmd/version.go]: Did the change..

**Comment by leecalcote** [comment]
Oh, I see that #793 is aimed at fixing this the additional help shown on display of version.

**Comment by ashiscs** [comment]
> Oh, I see that #793 is aimed at fixing this the additional help shown on display of version.



> Oh, I see that #793 is aimed at fixing this the additional help shown on display of version.

Yes Sir..

**Pull request merged**